CHANGELOG.md

```diff
@@ -1,5 +1,116 @@
 # Changelog
 
+## v6.0.0-rc - 2017-11-16
+
+### Breaking changes
+
+- Fix `-Verbose` to not override `$ErrorActionPreference`. (#5113)
+- Fix `Get-Item -LiteralPath a*b` to return error if `a*b` doesn't actually exist. (#5197)
+- Remove `AllScope` from most default aliases to reduce overhead on creating new scopes. (#5268)
+- Change `$OutputEncoding` to be `UTF8` without `BOM` rather than `ASCII`. (#5369)
```

Comment:
I think this should be: Change `$OutputEncoding` default to be ...

Reply:
Will change.